Photos/client/src/components/photos.jsx

```diff
@@ -0,0 +1,49 @@
+
+
+
+
+class Photos extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    return(
+
+      <div className = "outside" onClick={() => { this.props.modalHandler() }}>
+         <div className = 'container'>
+        
+        <div className = 'bigger'>
+        
+          <img src = {this.props.list[0].url}></img>
+        </div>
+  
+        <div className = 'containerdos'>
+          <div className = 'smaller'>
+            <img src = {this.props.list[1].url}></img>
+          </div>
+  
+          <div className = 'smaller'>
+            <img src = {this.props.list[2].url}></img>
+          </div>
+        </div>
+  
+        <div className = 'containerdos'>
```

Comment:
very happy/surprised to see containerdos, maybe consider renaming it to something that doesn't include numbers lol